docs/tutorial/index.rst

```diff
@@ -123,22 +131,64 @@ scheduler to aim to minimise the difference between room capacity and demand::
     Talk 1 at 15-Sep-2016 09:30 in Big
-    Talk 12 at 15-Sep-2016 09:30 in Small
+    Talk 2 at 15-Sep-2016 09:30 in Small
     Talk 3 at 15-Sep-2016 10:00 in Big
-    Talk 11 at 15-Sep-2016 10:00 in Small
-    Talk 4 at 15-Sep-2016 12:30 in Small
-    Talk 10 at 15-Sep-2016 12:30 in Big
-    Talk 2 at 15-Sep-2016 13:00 in Small
-    Talk 9 at 15-Sep-2016 13:00 in Big
-    Talk 7 at 16-Sep-2016 09:30 in Big
+    Talk 4 at 15-Sep-2016 10:00 in Small
+    Talk 8 at 15-Sep-2016 12:30 in Big
+    Talk 12 at 15-Sep-2016 12:30 in Small
+    Talk 5 at 15-Sep-2016 13:00 in Big
+    Talk 10 at 15-Sep-2016 13:00 in Small
+    Talk 11 at 16-Sep-2016 09:30 in Big
     Workshop 2 at 16-Sep-2016 09:30 in Small
-    Talk 6 at 16-Sep-2016 10:00 in Big
-    Talk 8 at 16-Sep-2016 12:30 in Big
+    Talk 9 at 16-Sep-2016 10:00 in Big
+    Talk 7 at 16-Sep-2016 12:30 in Big
     City tour at 16-Sep-2016 12:30 in Outside
-    Talk 5 at 16-Sep-2016 13:00 in Big
+    Talk 6 at 16-Sep-2016 13:00 in Big
     Workshop 1 at 16-Sep-2016 13:00 in Small
     Boardgames at 16-Sep-2016 13:00 in Outside
 
 
+
 We see that :code:`Talk 3` has moved to the bigger room but that all other
 constraints still hold.
+
+Coping with new information
+---------------------------
+
+This is fantastic! Our schedule has now been published and everyone is excited
+about the conference. However, as can often happen one of the speakers now
+informs us of a particular new constraints. For example, the speaker for
+:code:`Talk 7` is unable to speak on the second day.
+
+We can enter this new constraint::
+
+    >>> events[6].unavailability.extend(slots[9:])
+
+We can now solve the problem one more time from scratch which might result in a
+completely different schedule with a number of changes. We can however solve the
+problem with a new objective function which is to minimise the changes from the
+old schedule::
+
+
+    >>> func = objective_functions.number_of_changes
+    >>> schedule = scheduler.schedule(events, slots, objective_function=func, original_schedule=schedule)
+
+    >>> schedule = sorted(schedule, key=lambda item: item.slot.starts_at)
+    >>> for item in schedule:
+    ...     print(f"{item.event.name} at {item.slot.starts_at} in {item.slot.venue}")
+    Talk 1 at 15-Sep-2016 09:30 in Big
+    Talk 2 at 15-Sep-2016 09:30 in Small
+    Talk 3 at 15-Sep-2016 10:00 in Big
+    Talk 4 at 15-Sep-2016 10:00 in Small
+    Talk 8 at 15-Sep-2016 12:30 in Big
+    Talk 12 at 15-Sep-2016 12:30 in Small
+    Talk 7 at 15-Sep-2016 13:00 in Big
+    Talk 10 at 15-Sep-2016 13:00 in Small
+    Talk 11 at 16-Sep-2016 09:30 in Big
+    Workshop 2 at 16-Sep-2016 09:30 in Small
+    Talk 9 at 16-Sep-2016 10:00 in Big
+    Talk 5 at 16-Sep-2016 12:30 in Big
+    City tour at 16-Sep-2016 12:30 in Outside
+    Talk 6 at 16-Sep-2016 13:00 in Big
+    Workshop 1 at 16-Sep-2016 13:00 in Small
+    Boardgames at 16-Sep-2016 13:00 in Outside
```

Comment:
> I've just added some text that highlights the change a bit more but I don't specifically say what has changed (mainly because things are volatile and I'm trying to avoid writing in text something that might change)

Sure, makes sense.

> Some sort of schedule diff sounds good too!

I’ll raise an issue for it.